jekyll/_cci2/config-intro.adoc

```diff
@@ -0,0 +1,332 @@
+---
+version:
+- Cloud
+- Server v3.x
+- Server v2.x
+---
+= Configuration Introduction
+:page-layout: classic-docs
+:page-liquid:
+:page-description: "Learn how to get started with the CircleCI config.yml file."
+:icons: font
+:toc: macro
+:toc-title:
+
+This guide gets you started with the CircleCI `config.yml` file.
+
+toc::[]
+
+== Getting started with CircleCI config
+
+This guide covers the following topics:
+
+* How CircleCI finds and runs `config.yml`.
+* How you can use shell commands to perform actions.
+* How `config.yml` interacts with your code.
+* How to trigger a build.
+* How to use Docker containers to run your jobs in a specific environment.
+* How to use workflows to orchestrate your build.
+* How to add approval steps to your workflow.
+
+CircleCI believes in *configuration as code*. Consequently, the entire delivery process from build to deploy is orchestrated through a single file called `config.yml`. The `config.yml` file is located in a folder called `.circleci` at the top of your repo project. CircleCI uses the YAML syntax for config. See the <<writing-yaml#,Writing YAML>> document for guidance on the basics.
+
+== Part 1: Using the shell
+
+CircleCI provides an on-demand shell to run commands. In this first example, you will set up a build and execute a shell command.
+
+. If you have not done so already, <<first-steps#,sign up with CircleCI>> and select either GitHub or BitBucket.
```

Comment:
There is also the option to sign up by email. Can you make this more generic rather than specifying GH BB?

Reply:
👍